### DISCUSSION TITLE
Text Editors with decent Grammar Tools - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#72] [General]
# Text Editors with decent Grammar Tools - Alan Pope's blog

This is another blog post lifted wholesale out of my weekly newsletter. I do this when I get a bit verbose to keep the newsletter brief. The newsletter is becoming a blog incubator, which I’m okay with.
A reminder about that newsletter The newsletter is emailed every Friday - subscribe here, and is archived and available via RSS a few days later.
I talked a bit about the process of setting up the newsletter on episode 34 of Linux Matters Podcast.

https://popey.com/blog/2024/08/text-editors-with-grammar-tools/

**Comment by lewalkingdad** [comment]
Hello, 

I know you said no emacs and may be is the effect of when you have a hammer, everything looks like a nail, but emacs may check all the boxes: 
- emacs has data-recovery natively
- supports remote editing natively
- support grammar correction (via the traditional aspell or using some packages you can also have grammarly)

but I'm not a zealot, there is a cost of using emacs, it requires some (extensive) learning (good thing is that it has a bult-in, interactive documentation, so it does not have to be upfront) and depending on how flexible you want your emacs to be, some coding too.

**Reply by popey** [reply]
Heh! I wondered how long it might be before someone said, "I know you say no to X, but X." Congratulations for being the first.

I'm a bit contrarian and a refusenik here because I can't face the ignominy (I had to spell check that) to join the emacs (or vim) party so late. It's the same reason I refuse to use Arch Linux (despite having it on my SteamDeck). The supporters are so irritatingly passionate about it that I don't want them to think they've "won." Which is frankly stupid.

This is a 'me' problem, and I still appreciate your thoughtful comment.

**Comment by mhalano** [comment]
May I suggest LanguageTool? It's a nice tool: https://languagetool.org/
I saw there is a plugin for Sublime, but since I don't use Sublime I can't really test.

**Comment by reagle** [comment]
What did you decide? Nothing comes close to grammarly in terms of capability, and I would happily use it outside of Sublime Text if it had good keyboard shortcuts. Sadly it doesn't, so I grumble as I do a lot of pointing and clicking.
